src/docs/asciidoc/common/development/hot-reload.adoc

```diff
@@ -0,0 +1,353 @@
+:relative-path: ../../
+include::{docdir}/variables.adoc[]
+
+In order to improve productivity, ZetaPush provides *hot reload* feature.
```

Comment:
Instead of this very verbose explanation and example, I would have simply described the livereload feature as : 

"If you know [nodemon](https://nodemon.io/), you know what is **hot-reload**.
ZetaPush monitor for any changes in your _custom cloud services_ code and **automatically reload your worker**. For the front part, the local HTTP server doesn't use cache locally, it always **serves the latest version of your code**. 

To test this feature, you can reuse the *HelloWorldAsCustomCloudService*, change the return value of `helloWorld` method ***after starting*** your worker, and observe that when you click the button at front side, the message in the console also changes."

The problem of this part is the code example, it is very heavy et long compared to what it helps the developer to understand.